packages/vapor/resources/icons/svg/slack.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg"><path d="M3.22815249 9.70948192c0 .88836758-.7257087 1.61407628-1.61407624 1.61407628C.7257087 11.3235582 0 10.5978495 0 9.70948192c0-.88836755.7257087-1.61407625 1.61407625-1.61407625h1.61407624v1.61407625zM4.04144673 9.70948192c0-.88836755.7257087-1.61407625 1.61407624-1.61407625.88836755 0 1.61407625.7257087 1.61407625 1.61407625v4.04144668c0 .8883676-.7257087 1.6140763-1.61407625 1.6140763-.88836754 0-1.61407624-.7257087-1.61407624-1.6140763V9.70948192z" fill="#E01E5A"/><path d="M5.65552297 3.22815249c-.88836754 0-1.61407624-.7257087-1.61407624-1.61407624C4.04144673.7257087 4.76715543 0 5.65552297 0c.88836755 0 1.61407625.7257087 1.61407625 1.61407625v1.61407624H5.65552297zM5.65552297 4.04144673c.88836755 0 1.61407625.7257087 1.61407625 1.61407624 0 .88836755-.7257087 1.61407625-1.61407625 1.61407625H1.61407625C.7257087 7.26959922 0 6.54389052 0 5.65552297c0-.88836754.7257087-1.61407624 1.61407625-1.61407624h4.04144672z" fill="#36C5F0"/><path d="M12.1368524 5.65552297c0-.88836754.7257087-1.61407624 1.6140762-1.61407624.8883676 0 1.6140763.7257087 1.6140763 1.61407624 0 .88836755-.7257087 1.61407625-1.6140763 1.61407625h-1.6140762V5.65552297zM11.3235582 5.65552297c0 .88836755-.7257087 1.61407625-1.61407628 1.61407625-.88836755 0-1.61407625-.7257087-1.61407625-1.61407625V1.61407625C8.09540567.7257087 8.82111437 0 9.70948192 0c.88836758 0 1.61407628.7257087 1.61407628 1.61407625v4.04144672z" fill="#2EB67D"/><path d="M9.70948192 12.1368524c.88836758 0 1.61407628.7257087 1.61407628 1.6140762 0 .8883676-.7257087 1.6140763-1.61407628 1.6140763-.88836755 0-1.61407625-.7257087-1.61407625-1.6140763v-1.6140762h1.61407625zM9.70948192 11.3235582c-.88836755 0-1.61407625-.7257087-1.61407625-1.61407628 0-.88836755.7257087-1.61407625 1.61407625-1.61407625h4.04144668c.8883676 0 1.6140763.7257087 1.6140763 1.61407625 0 .88836758-.7257087 1.61407628-1.6140763 1.61407628H9.70948192z" fill="#ECB22E"/></svg>
```

Comment:
I think you are missing the viewbox on the svg element. I tried an online tool and I came up with this:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 15.365 15.365">
  <path d="M3.22815249 9.70948192c0 .88836758-.7257087 1.61407628-1.61407624 1.61407628C.7257087 11.3235582 0 10.5978495 0 9.70948192c0-.88836755.7257087-1.61407625 1.61407625-1.61407625h1.61407624v1.61407625zM4.04144673 9.70948192c0-.88836755.7257087-1.61407625 1.61407624-1.61407625.88836755 0 1.61407625.7257087 1.61407625 1.61407625v4.04144668c0 .8883676-.7257087 1.6140763-1.61407625 1.6140763-.88836754 0-1.61407624-.7257087-1.61407624-1.6140763V9.70948192z" fill="#E01E5A"/>
  <path d="M5.65552297 3.22815249c-.88836754 0-1.61407624-.7257087-1.61407624-1.61407624C4.04144673.7257087 4.76715543 0 5.65552297 0c.88836755 0 1.61407625.7257087 1.61407625 1.61407625v1.61407624H5.65552297zM5.65552297 4.04144673c.88836755 0 1.61407625.7257087 1.61407625 1.61407624 0 .88836755-.7257087 1.61407625-1.61407625 1.61407625H1.61407625C.7257087 7.26959922 0 6.54389052 0 5.65552297c0-.88836754.7257087-1.61407624 1.61407625-1.61407624h4.04144672z" fill="#36C5F0"/>
  <path d="M12.1368524 5.65552297c0-.88836754.7257087-1.61407624 1.6140762-1.61407624.8883676 0 1.6140763.7257087 1.6140763 1.61407624 0 .88836755-.7257087 1.61407625-1.6140763 1.61407625h-1.6140762V5.65552297zM11.3235582 5.65552297c0 .88836755-.7257087 1.61407625-1.61407628 1.61407625-.88836755 0-1.61407625-.7257087-1.61407625-1.61407625V1.61407625C8.09540567.7257087 8.82111437 0 9.70948192 0c.88836758 0 1.61407628.7257087 1.61407628 1.61407625v4.04144672z" fill="#2EB67D"/>
  <path d="M9.70948192 12.1368524c.88836758 0 1.61407628.7257087 1.61407628 1.6140762 0 .8883676-.7257087 1.6140763-1.61407628 1.6140763-.88836755 0-1.61407625-.7257087-1.61407625-1.6140763v-1.6140762h1.61407625zM9.70948192 11.3235582c-.88836755 0-1.61407625-.7257087-1.61407625-1.61407628 0-.88836755.7257087-1.61407625 1.61407625-1.61407625h4.04144668c.8883676 0 1.6140763.7257087 1.6140763 1.61407625 0 .88836758-.7257087 1.61407628-1.6140763 1.61407628H9.70948192z" fill="#ECB22E"/>
</svg>
```

It makes the icon responsive

Reply:
TIL thx! will do that